py/ringbuf.c

```diff
@@ -71,3 +71,49 @@ int ringbuf_put16(ringbuf_t *r, uint16_t v) {
     r->iput = iput_b;
     return 0;
 }
+
+#include <string.h>
```

Comment:
I just realised that these changes (to py/ringbuf.c and py/ringbuf.h) should be in a separate commit, because they are adding general functionality to a shared piece of code, ie they are independent to ESP NOW.

One way to do this without much effort is (when on the branch for this PR):
```
$ git checkout HEAD^ py/ringbuf.[ch]
$ git commit -m revert
$ git checkout HEAD^ py/ringbuf.[ch]
$ git commit -m "py/ringbuf: Implement read/write functions."
```
Then squash the "revert" commit into the main big commit.

And this include here should go to the top of the file.